Vagrantfile

```diff
@@ -2,7 +2,7 @@
 # vi: set ft=ruby :
 
 INVENTORY_FILE = "inventory.vagrant"
-MEMORY = 384
+MEMORY = 1024
```

Comment:
I have first tested with 512 and still wasn't enough. If there is anyother option I would be happy take it.